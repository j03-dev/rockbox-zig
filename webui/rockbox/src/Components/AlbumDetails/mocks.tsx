export const tracks = [
  {
    id: "1",
    trackNumber: 1,
    title: "Gasoline (Live)",
    artist: "Seether",
    time: "2:57",
  },
  {
    id: "2",
    trackNumber: 2,
    title: "Driven Under (Live)",
    artist: "Seether",
    time: "4:58",
  },
  {
    id: "3",
    trackNumber: 3,
    title: "Diseased (Live)",
    artist: "Seether",
    time: "3:46",
  },
  {
    id: "4",
    trackNumber: 4,
    title: "Truth (Live)",
    artist: "Seether",
    time: "5:15",
  },
  {
    id: "5",
    trackNumber: 5,
    title: "Immortality (Live)",
    artist: "Seether",
    time: "5:02",
  },
  {
    id: "6",
    trackNumber: 6,
    title: "Tied My Hands (Live)",
    artist: "Seether",
    time: "5:16",
  },
  {
    id: "7",
    trackNumber: 7,
    title: "Sympathetic (Live)",
    artist: "Seether",
    time: "4:12",
  },
  {
    id: "8",
    trackNumber: 8,
    title: "Fine Again (Live)",
    artist: "Seether",
    time: "5:05",
  },
  {
    id: "9",
    trackNumber: 9,
    title: "Broken (Live)",
    artist: "Seether",
    time: "4:17",
  },
  {
    id: "10",
    trackNumber: 10,
    title: "The Gift (Live)",
    artist: "Seether",
    time: "5:36",
  },
  {
    id: "11",
    trackNumber: 11,
    title: "Remedy (Live)",
    artist: "Seether",
    time: "3:40",
  },
  {
    id: "12",
    trackNumber: 12,
    title: "Plastci Man (Live)",
    artist: "Seether",
    time: "3:34",
  },
  {
    id: "13",
    trackNumber: 13,
    title: "The Gist (Alternate Mix)",
    artist: "Seether",
    time: "3:24",
  },
];
