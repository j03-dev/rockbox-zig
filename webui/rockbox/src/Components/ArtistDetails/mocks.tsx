export const tracks = [
  {
    id: "1",
    trackNumber: 1,
    title: "Get Lucky",
    artist: "Daft Punk",
    album: "Random Access Memories",
    albumArt:
      "https://resources.tidal.com/images/09b59e6e/717e/43e3/b2e2/d2a153c24775/320x320.jpg",
    time: "6:10",
    artistId: "1",
    albumId: "1",
  },
  {
    id: "2",
    trackNumber: 2,
    title: "Instant Crush",
    artist: "Daft Punk",
    album: "Random Access Memories",
    albumArt:
      "https://resources.tidal.com/images/09b59e6e/717e/43e3/b2e2/d2a153c24775/320x320.jpg",
    time: "5:38",
    artistId: "1",
    albumId: "2",
  },
  {
    id: "3",
    trackNumber: 2,
    title: "Arround the World",
    artist: "Daft Punk",
    album: "Homework",
    albumArt:
      "https://resources.tidal.com/images/f853861c/0c5f/4e73/b608/eeb00618fe6f/320x320.jpg",
    time: "7:10",
    artistId: "1",
    albumId: "3",
  },
];

export const albums = [
  {
    id: "1",
    title: "Random Access Memories",
    artist: "Daft Punk",
    albumArt:
      "https://resources.tidal.com/images/09b59e6e/717e/43e3/b2e2/d2a153c24775/320x320.jpg",
    year: 2013,
  },
  {
    id: "2",
    title: "Tron: Legacy",
    artist: "Daft Punk",
    albumArt:
      "https://resources.tidal.com/images/866bb671/5ec8/4b45/8a60/50e51f5dbf10/320x320.jpg",
    year: 2010,
  },
  {
    id: "3",
    title: "Homework",
    artist: "Daft Punk",
    albumArt:
      "https://resources.tidal.com/images/f853861c/0c5f/4e73/b608/eeb00618fe6f/320x320.jpg",
    year: 2001,
  },
];
