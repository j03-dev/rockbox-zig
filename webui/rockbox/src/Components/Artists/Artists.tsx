/* eslint-disable @typescript-eslint/no-explicit-any */
import { FC } from "react";
import { Cell, Grid } from "baseui/layout-grid";
import Sidebar from "../Sidebar";
import ControlBar from "../ControlBar";
import {
  ArtistCover,
  ArtistName,
  Container,
  MainView,
  NoArtistCover,
  Scrollable,
  Title,
} from "./styles";
import Artist from "../Icons/Artist";
import { Link } from "react-router-dom";

export type ArtistsProps = {
  artists: any[];
  onClickArtist: (artist: any) => void;
  onFilter: (filter: string) => void;
};

const Artists: FC<ArtistsProps> = (props) => {
  const { onClickArtist, artists } = props;
  return (
    <Container>
      <Sidebar active="artists" />
      <MainView>
        <ControlBar />
        <Scrollable>
          <Title>Artists</Title>
          <div style={{ marginBottom: 100 }}>
            <Grid
              gridColumns={[2, 3, 4]}
              gridMargins={[18, 18, 18]}
              gridGutters={[10, 10, 10]}
            >
              {artists.map((item) => (
                <Cell key={item.id} align={"center"}>
                  <Link
                    to={`/artists/${item.id}`}
                    style={{ textDecoration: "none" }}
                  >
                    {item.cover && (
                      <ArtistCover
                        src={item.cover}
                        onClick={() => onClickArtist(item)}
                      />
                    )}
                    {!item.cover && (
                      <NoArtistCover onClick={() => onClickArtist(item)}>
                        <Artist width={75} height={75} color="#a4a3a3" />
                      </NoArtistCover>
                    )}
                    <ArtistName>{item.name}</ArtistName>
                  </Link>
                </Cell>
              ))}
            </Grid>
          </div>
        </Scrollable>
      </MainView>
    </Container>
  );
};

export default Artists;
