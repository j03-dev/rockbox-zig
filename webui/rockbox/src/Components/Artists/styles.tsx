import styled from "@emotion/styled";

export const Container = styled.div`
  display: flex;
  flex-direction: row;
  width: 100%;
  height: 100%;
`;

export const MainView = styled.div`
  display: flex;
  flex: 1;
  flex-direction: column;
`;

export const Title = styled.div`
  font-size: 24px;
  font-family: RockfordSansMedium;
  margin: auto;
  margin-bottom: 20px;
  padding-left: 20px;
  padding-right: 20px;
`;

export const Scrollable = styled.div`
  height: calc(100vh - 60px);
  overflow-y: auto;
`;

export const Wrapper = styled.div`
  margin-top: 34px;
`;

export const ArtistCover = styled.img`
  width: 194px;
  height: 194px;
  border-radius: 97px;
  cursor: pointer;
`;

export const NoArtistCover = styled.div`
  width: 194px;
  height: 194px;
  border-radius: 97px;
  cursor: pointer;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #f3f3f3b9;
`;

export const ArtistName = styled.div`
  font-size: 14px;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  cursor: pointer;
  margin-top: 20px;
  margin-bottom: 18px;
  text-align: center;
  width: 194px;
  color: #000;
`;
