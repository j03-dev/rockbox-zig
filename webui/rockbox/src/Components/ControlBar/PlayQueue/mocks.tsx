/* eslint-disable no-loss-of-precision */

export const nextTracks = [
  {
    id: "5ce25a3dadabc01f94130d806294a296",
    title: "Can I",
    artist: "Kodak Black",
    album: "Lil Big Pac",
    cover:
      "https://resources.tidal.com/images/f36fa0ef/fe50/4c54/8b97/3c9f4f72f8e8/320x320.jpg",
    duration: 216083.99963378906,
    artistId: "c38e31502394622bbdfd3d82f6911eb3",
    albumId: "229251493",
  },
  {
    id: "c2754b1edfcb8fb4dd34d94f4e42b622",
    title: "Too Many Years",
    artist: "Kodak Black, PnB Rock",
    album: "Lil Big Pac",
    cover:
      "https://resources.tidal.com/images/f36fa0ef/fe50/4c54/8b97/3c9f4f72f8e8/320x320.jpg",
    duration: 196486.99951171875,
    artistId: "a01003e6ac321328ad1e208bf41206f0",
    albumId: "229251493",
  },
  {
    id: "89ab06746b363d3295a2bc297339c5be",
    title: "ALL THE TIME HIGH",
    artist: "Juicy J featuring Kaash Paige",
    album: "THE HUSTLE STILL CONTINUES (Deluxe)",
    cover:
      "https://resources.tidal.com/images/aeb814c6/7767/421f/a121/46fdc0895c53/320x320.jpg",
    duration: 120141.99829101562,
    artistId: "c96c630a2fd2edd0eec175c4f65e0329",
    albumId: "229251493",
  },
];

export const previousTracks = [
  {
    id: "93b7e61e27b4e5cab4c6ac4d45a4dc9f",
    title: "Sobriety Sucks",
    artist: "Yelawolf",
    album: "Trunk Muzic Returns (Deluxe Edition)",
    cover:
      "https://resources.tidal.com/images/3b8d6b45/b3e4/4f0c/a69d/2f31a1fceadf/320x320.jpg",
    duration: 255001.00708007812,
    artistId: "052dcb0741a91da6d26a35600ec46cd9",
  },
];
