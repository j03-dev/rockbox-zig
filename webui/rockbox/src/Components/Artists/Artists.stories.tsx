import type { Meta, StoryObj } from "@storybook/react";

import Artists from "./Artists";
import { fn } from "@storybook/test";
import { artists } from "./mocks";

// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
const meta = {
  title: "Components/Artists",
  component: Artists,
  // This component will have an automatically generated Autodocs entry: https://storybook.js.org/docs/writing-docs/autodocs
  tags: ["autodocs"],
  // More on argTypes: https://storybook.js.org/docs/api/argtypes
} satisfies Meta<typeof Artists>;

export default meta;
type Story = StoryObj<typeof meta>;

// More on writing stories with args: https://storybook.js.org/docs/writing-stories/args
export const Default: Story = {
  args: {
    onClickArtist: fn(),
    onFilter: fn(),
    artists,
  },
};
