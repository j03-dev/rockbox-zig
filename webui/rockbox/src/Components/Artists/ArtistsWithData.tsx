import { FC } from "react";
import { artists } from "./mocks";
import Artists from "./Artists";

const ArtistsWithData: FC = () => {
  return (
    <Artists onFilter={() => {}} onClickArtist={() => {}} artists={artists} />
  );
};

export default ArtistsWithData;
