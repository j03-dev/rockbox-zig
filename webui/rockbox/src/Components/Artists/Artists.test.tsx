import { render } from "@testing-library/react";
import { vi } from "vitest";
import Artists from "./Artists";
import { artists } from "./mocks";
import Providers from "../../Providers";
import { MemoryRouter } from "react-router-dom";
import { MockedProvider } from "@apollo/client/testing";
import { mocks } from "../../mocks";

describe("Artists", () => {
  it("should render", () => {
    const { container } = render(
      <MemoryRouter initialEntries={["/"]}>
        <MockedProvider mocks={mocks}>
          <Providers>
            <Artists
              artists={artists}
              onClickArtist={vi.fn()}
              onFilter={vi.fn()}
            />
          </Providers>
        </MockedProvider>
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
