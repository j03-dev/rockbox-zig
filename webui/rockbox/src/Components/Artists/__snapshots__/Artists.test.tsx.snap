// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Artists > should render 1`] = `
<div>
  <div
    class=""
  >
    <div
      class="css-e1989k"
    >
      <div
        class="css-1fkc5kv"
      >
        <a
          class="css-tujm8o"
          color="initial"
          href="/albums"
        >
          <svg
            aria-hidden="true"
            class="StyledIconBase-sc-ea9ulj-0 dmvaRK"
            color="initial"
            fill="currentColor"
            focusable="false"
            height="20"
            style="margin-right: 6px;"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm0 18a8 8 0 1 1 8-8 8 8 0 0 1-8 8z"
            />
            <path
              d="M12 8a4 4 0 1 0 4 4 4 4 0 0 0-4-4zm0 6a2 2 0 1 1 2-2 2 2 0 0 1-2 2z"
            />
          </svg>
          <div>
            Albums
          </div>
        </a>
        <a
          class="css-8v05qj"
          color="#fe099c"
          href="/artists"
        >
          <svg
            height="20"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M20 4.22a5.67 5.67 0 0 0-9.68 4.57l-8 9.79 3.3 3.3 9.79-8c.18 0 .36.05.55.05a5.7 5.7 0 0 0 4-9.73ZM5.74 19.86l-1.38-1.38 6.44-7.89a5.48 5.48 0 0 0 2.83 2.84Zm13.21-8.65a4.2 4.2 0 1 1 0-5.94 4.17 4.17 0 0 1 0 5.95Z"
              fill="#fe099c"
            />
          </svg>
          <div
            style="margin-left: 6px;"
          >
            Artists
          </div>
        </a>
        <a
          class="css-tujm8o"
          color="initial"
          href="/tracks"
        >
          <svg
            height="20"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8.1 4.65v11.26a3.45 3.45 0 1 0 1.5 2.84V5.85l10.2-2.36v10.62A3.45 3.45 0 1 0 21.3 17V1.61Zm-2 16a2 2 0 1 1 2-2 2 2 0 0 1-1.95 2.05Zm11.7-1.8a1.95 1.95 0 1 1 2-1.85 2 2 0 0 1-1.95 1.9Z"
              fill="initial"
            />
          </svg>
          <div
            style="margin-left: 6px;"
          >
            Songs
          </div>
        </a>
        <a
          class="css-tujm8o"
          color="initial"
          href="/files"
        >
          <svg
            aria-hidden="true"
            class="StyledIconBase-sc-ea9ulj-0 dmvaRK"
            color="initial"
            fill="none"
            focusable="false"
            height="19"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            style="margin-right: 6px;"
            viewBox="0 0 24 24"
            width="19"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="22"
              x2="2"
              y1="12"
              y2="12"
            />
            <path
              d="M5.45 5.11 2 12v6a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2v-6l-3.45-6.89A2 2 0 0 0 16.76 4H7.24a2 2 0 0 0-1.79 1.11z"
            />
            <line
              x1="6"
              x2="6.01"
              y1="16"
              y2="16"
            />
            <line
              x1="10"
              x2="10.01"
              y1="16"
              y2="16"
            />
          </svg>
          <div>
            Files
          </div>
        </a>
      </div>
      <div
        class="css-1qxls6i"
      >
        <div
          class="css-1dlrxtg"
        >
          <div
            class="css-1h69jy4"
          >
            <div
              class="css-1ytum11"
            >
              <button
                class="css-10asy2d"
              >
                <svg
                  data-test="icon-PlaybackShuffle"
                  fill="none"
                  height="16"
                  viewBox="0 0 24 24"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M16.2 7h2.85l-1.3 1.3c-.4.4-.4 1 0 1.4.25.2.5.3.75.3s.5-.1.7-.3l3-3 .15-.15.05-.05c0-.025.013-.037.025-.05.012-.013.025-.025.025-.05 0-.021 0-.033.004-.044.005-.015.017-.027.046-.056 0-.021 0-.033.004-.044.005-.015.017-.027.046-.056.05-.15.05-.25 0-.4 0-.05 0-.05-.05-.1 0-.021 0-.033-.004-.044-.005-.015-.017-.027-.046-.056 0-.025-.012-.037-.025-.05-.012-.013-.025-.025-.025-.05 0 0 0-.05-.05-.05l-.15-.15-3-3c-.4-.4-1-.4-1.4 0-.4.4-.4 1 0 1.4L19.1 5h-2.85c-1.75 0-3.3.9-4.25 2.35L6.85 15.6c-.55.9-1.5 1.4-2.55 1.4H2.5c-.55 0-1 .45-1 1s.45 1 1 1h1.8c1.75 0 3.3-.9 4.25-2.35L13.7 8.4c.5-.85 1.5-1.4 2.5-1.4ZM2.5 7h1.8c1.05 0 2 .5 2.55 1.4l.45.75 1.2-1.9C7.55 5.8 6 4.95 4.3 4.95H2.5c-.55 0-1 .45-1 1 0 .6.45 1.05 1 1.05Zm19.95 11.2v.05c0 .05 0 .1-.05.1v.05c-.05.1-.1.2-.2.3l-3 3c-.2.2-.45.3-.7.3-.25 0-.5-.1-.75-.3-.4-.4-.4-1 0-1.4l1.3-1.3H16.2c-1.7 0-3.25-.85-4.2-2.3l1.25-1.9.45.75c.55.9 1.5 1.4 2.55 1.4h2.85l-1.3-1.3c-.4-.4-.4-1 0-1.4.4-.4 1-.4 1.4 0l3 3c.1.1.15.2.2.3v.05c.05.05.05.1.05.15v.05c.05.15.05.25 0 .4Z"
                    fill="#000"
                    fill-rule="evenodd"
                  />
                </svg>
              </button>
              <button
                class="css-10asy2d"
              >
                <svg
                  fill="none"
                  height="18"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="ionicon"
                    d="M3.55 2c.304 0 .55.243.55.542V7.16l8.001-4.716a1.224 1.224 0 0 1 1.23-.015c.413.23.669.677.669 1.162v9.818c0 .486-.256.932-.669 1.162-.382.216-.853.21-1.23-.015L4.1 9.84v4.618a.546.546 0 0 1-.55.542.546.546 0 0 1-.55-.542V2.542C3 2.243 3.246 2 3.55 2Z"
                    style="fill: #000;"
                  />
                </svg>
              </button>
              <button
                class="css-10asy2d"
              >
                <svg
                  height="32"
                  viewBox="0 0 512 512"
                  width="32"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title>
                    Pause
                  </title>
                  <path
                    d="M208 432h-48a16 16 0 0 1-16-16V96a16 16 0 0 1 16-16h48a16 16 0 0 1 16 16v320a16 16 0 0 1-16 16zm144 0h-48a16 16 0 0 1-16-16V96a16 16 0 0 1 16-16h48a16 16 0 0 1 16 16v320a16 16 0 0 1-16 16z"
                    style="fill: #000;"
                  />
                </svg>
              </button>
              <button
                class="css-10asy2d"
              >
                <svg
                  fill="none"
                  height="18"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    class="ionicon"
                    d="M14.45 2a.546.546 0 0 0-.55.542V7.16L5.899 2.444a1.224 1.224 0 0 0-1.23-.015C4.256 2.659 4 3.105 4 3.591v9.818c0 .486.256.932.669 1.162.382.216.853.21 1.23-.015L13.9 9.84v4.618c0 .299.246.542.55.542.304 0 .55-.243.55-.542V2.542A.546.546 0 0 0 14.45 2Z"
                    style="fill: #000;"
                  />
                </svg>
              </button>
              <button
                class="css-10asy2d"
              >
                <svg
                  data-test="icon-PlaybackInactiveRepeat"
                  fill="none"
                  height="16"
                  viewBox="0 0 24 24"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M16.293 1.293a1 1 0 0 1 1.414 0l3 3a1 1 0 0 1 0 1.414l-3 3a1 1 0 1 1-1.414-1.414L17.586 6H8.5c-3.298 0-6 2.702-6 6a1 1 0 1 1-2 0c0-4.402 3.598-8 8-8h9.086l-1.293-1.293a1 1 0 0 1 0-1.414ZM23.5 12a1 1 0 1 0-2 0c0 3.298-2.702 6-6 6H6.414l1.293-1.293a1 1 0 1 0-1.414-1.414l-3 3a1 1 0 0 0 0 1.414l3 3a1 1 0 0 0 1.414-1.414L6.414 20H15.5c4.402 0 8-3.598 8-8Z"
                    fill="#000"
                    fill-rule="evenodd"
                  />
                </svg>
              </button>
            </div>
          </div>
          <div
            class="css-frud38"
          >
            <img
              alt="Album cover"
              class="css-31cka"
              src="https://resources.tidal.com/images/fe6787d5/4ba5/4d3e/8576/48943ee6a768/320x320.jpg"
            />
            <div
              class="css-rbpzuh"
            >
              <div
                class="css-1cojt9j"
              >
                Drankin N Smokin
              </div>
              <div
                style="display: flex; flex-direction: row; align-items: center; justify-content: space-between;"
              >
                <div
                  class="css-1jnbdma"
                >
                  02:03
                </div>
                <div
                  class="css-bdl275"
                >
                  Future, Lil Uzi Vert
                  <span
                    class="css-c9cgcr"
                  >
                    -
                  </span>
                  Pluto x Baby Pluto (Deluxe)
                </div>
                <div
                  class="css-1jnbdma"
                >
                  04:15
                </div>
              </div>
              <div
                class="css-1n7uud4"
              >
                <div
                  aria-label="48% Loaded"
                  aria-valuemax="100"
                  aria-valuemin="0"
                  aria-valuenow="48.3219508605943"
                  class="ae"
                  data-baseweb="progress-bar"
                  role="progressbar"
                >
                  <div
                    class="af ag ah ai aj"
                  >
                    <div
                      class="ak al am an ao ap aq ar"
                    >
                      <div
                        class="ak al am an as at ae au av"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-nb4c1y"
          >
            <button
              class="css-10asy2d"
            >
              <svg
                aria-hidden="true"
                class="StyledIconBase-sc-ea9ulj-0 dmvaRK"
                fill="currentColor"
                focusable="false"
                height="21"
                viewBox="0 0 20 20"
                width="21"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M14.4 9H8.6c-.552 0-.6.447-.6 1s.048 1 .6 1h5.8c.552 0 .6-.447.6-1s-.048-1-.6-1zm2 5H8.6c-.552 0-.6.447-.6 1s.048 1 .6 1h7.8c.552 0 .6-.447.6-1s-.048-1-.6-1zM8.6 6h7.8c.552 0 .6-.447.6-1s-.048-1-.6-1H8.6c-.552 0-.6.447-.6 1s.048 1 .6 1zM5.4 9H3.6c-.552 0-.6.447-.6 1s.048 1 .6 1h1.8c.552 0 .6-.447.6-1s-.048-1-.6-1zm0 5H3.6c-.552 0-.6.447-.6 1s.048 1 .6 1h1.8c.552 0 .6-.447.6-1s-.048-1-.6-1zm0-10H3.6c-.552 0-.6.447-.6 1s.048 1 .6 1h1.8c.552 0 .6-.447.6-1s-.048-1-.6-1z"
                />
              </svg>
            </button>
          </div>
        </div>
        <div
          class="css-phivbe"
        >
          <div
            class="css-1pv8gyt"
          >
            Artists
          </div>
          <div
            style="margin-bottom: 100px;"
          >
            <div
              class="aw ax"
            >
              <div
                class="ay af az ax b0 b1 b2 b3 b4 b5 b6 b7 b8 b9 ba bb bc bd be bf"
              >
                <div
                  class="ay ae bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz c0 c1 c2 c3 c4 c5"
                >
                  <a
                    href="/artists/1"
                    style="text-decoration: none;"
                  >
                    <img
                      class="css-xfzo3c"
                      src="https://resources.tidal.com/images/9be9e74c/1b06/479a/a79e/5c62733d74ac/320x320.jpg"
                    />
                    <div
                      class="css-165pfvn"
                    >
                      Red Hot Chilli Peppers
                    </div>
                  </a>
                </div>
                <div
                  class="ay ae bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz c0 c1 c2 c3 c4 c5"
                >
                  <a
                    href="/artists/2"
                    style="text-decoration: none;"
                  >
                    <img
                      class="css-xfzo3c"
                      src="https://resources.tidal.com/images/f87d9afc/075e/43f4/bbbc/7770b46cb8aa/320x320.jpg"
                    />
                    <div
                      class="css-165pfvn"
                    >
                      Daft Punk
                    </div>
                  </a>
                </div>
                <div
                  class="ay ae bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz c0 c1 c2 c3 c4 c5"
                >
                  <a
                    href="/artists/3"
                    style="text-decoration: none;"
                  >
                    <img
                      class="css-xfzo3c"
                      src="https://resources.tidal.com/images/c3672adf/ca16/4db3/a375/64904d1f7d39/320x320.jpg"
                    />
                    <div
                      class="css-165pfvn"
                    >
                      Slipknot
                    </div>
                  </a>
                </div>
                <div
                  class="ay ae bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz c0 c1 c2 c3 c4 c5"
                >
                  <a
                    href="/artists/4"
                    style="text-decoration: none;"
                  >
                    <img
                      class="css-xfzo3c"
                      src="https://resources.tidal.com/images/2021e89d/f4bd/4d41/9af7/da47dc7564e8/320x320.jpg"
                    />
                    <div
                      class="css-165pfvn"
                    >
                      Big K.R.I.T.
                    </div>
                  </a>
                </div>
                <div
                  class="ay ae bg bh bi bj bk bl bm bn bo bp bq br bs bt bu bv bw bx by bz c0 c1 c2 c3 c4 c5"
                >
                  <a
                    href="/artists/4"
                    style="text-decoration: none;"
                  >
                    <div
                      class="css-qsoif3"
                    >
                      <svg
                        height="75"
                        viewBox="0 0 24 24"
                        width="75"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M20 4.22a5.67 5.67 0 0 0-9.68 4.57l-8 9.79 3.3 3.3 9.79-8c.18 0 .36.05.55.05a5.7 5.7 0 0 0 4-9.73ZM5.74 19.86l-1.38-1.38 6.44-7.89a5.48 5.48 0 0 0 2.83 2.84Zm13.21-8.65a4.2 4.2 0 1 1 0-5.94 4.17 4.17 0 0 1 0 5.95Z"
                          fill="#a4a3a3"
                        />
                      </svg>
                    </div>
                    <div
                      class="css-165pfvn"
                    >
                      Oasis
                    </div>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class=""
  />
</div>
`;
